service/routes.py

```diff
@@ -9,17 +9,11 @@
 PUT /inventories/{ID}/in-stock - update the in_stock attribute of Inventory model to True
 PUT /inventories/{ID}/out-of-stock - update the in_stock attribute of Inventory model to False
 """
-import os
```

Comment:
PyLint was complaining about these imports as they were unused. We can add them back in if we end up needing them later on.